### DISCUSSION TITLE
In `editbookmarkspopup_expanded.uc.js`, add URL editbox to allow URL modification on bookmark popup

### DISCUSSION THREAD

**Discussion by jaejunks** [Aris-t2/CustomJSforFx#107] [Ideas]
If possible for the `editbookmarkspopup_expanded.uc.js` script, add an editbox on the bookmark popup for modifying the bookmark URL.

Reason is that, some sites redirect the root URL e.g. `https://site.com/` to e.g. `https://site.com/main.php`. But later changed it to e.g. `https://site.com/home.php`. This makes bookmark which pointed to `https://site.com/main.php` became no longer valid.

So, when adding bookmark to sites like above mentioned, I want to be able to bookmark the site as `https://site.com/` instead of `https://site.com/home.php`.

Thanks in advance.

**Comment by Aris-t2** [comment]
This is far beyond what this script was for or can do.
